lib/command.ts

```diff
@@ -261,6 +265,10 @@ export function command(
       // as a short hand, an object can instead be provided, specifying
       // the options that a command takes.
       const innerYargs = yargs.reset(parsed.aliases);
+      // A null command indicates we are running the default command,
```

Comment:
@OsmanAltun I think this does the trick, we want to `unfreeze` if we've called `reset` (which will put the usage output in the original state), and if we're running the default command.